lib/rucio/web/rest/flaskapi/v1/archive.py

```diff
@@ -42,27 +46,32 @@ class Archive(MethodView):
     """ REST APIs for archive. """
 
     @check_accept_header_wrapper_flask(['application/x-json-stream'])
-    def get(self, scope, name):
+    def get(self, scope_name):
```

Comment:
why this change?

Reply:
because schemas can configure an arbitary `SCOPE_NAME_REGEX`[(1)] and this is backwards-compatible.

[(1)]: https://github.com/rucio/rucio/blob/4ab624fd77949d97a3498e2d31ce58bfd6eef135/lib/rucio/common/schema/cms.py#L391

Reply:
ah right, I forgot about the schema.

Reply:
I am all open for a better solution, if you have any :D

Reply:
not really. I just missed the step with the schema matching. all good from my side.